packages/Confirmation/src/components/TriggerButton/TriggerButton.js

```diff
@@ -20,7 +20,7 @@ const TriggerButton = props => {
       data-pka-anchor="button"
       ref={triggerRef}
       aria-expanded={isConfirmOpen}
-      aria-haspopup="true"
+      aria-haspopup
```

Comment:
This change and the one to `OverflowMenu` are patch updates and while we _could_ include them in the `CHANGELOG`, I think they're so small and insignificant that we can omit that update.